.github/workflows/techlabblog-deploy-dev.yml

```
@@ -0,0 +1,50 @@
+name: Techlab Blog | Deploy Dev
```

```suggestion
name: Techlab Blog | Dev | Deploy
```

or `Techlab Blog | Deploy | Dev`